src/vsgQt/ViewerWindow.cpp

```diff
@@ -62,9 +66,17 @@ void ViewerWindow::cleanup()
     if (windowAdapter)
     {
 #if QT_HAS_VULKAN_SUPPORT
-        if (surfaceType() == QSurface::VulkanSurface)
+        auto platform = qGuiApp->platformName();
```

Comment:
I am confused why you have added initialize functions to a cleanup() function.

Reply:
Oh, I'm sorry I made a mistake when I splitted the PR. There were few changes and I decided to split it manually from a clean repository.